services/user-services.js

```diff
@@ -0,0 +1,56 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
+const userDao = require('../models/user.dao');
+
+const { validateEmail }  = require('../../util/validators');
+
+
+
+const signUp = async(email, password, firstname, lastname, skintype) => {
+ 
+   await validateEmail(email)
+   
+   const user = await userDao.getuserByEmail(email);
+  
+   if (user){
```

Comment:
마이너하지만 해당 부분은 지속 언급드린 사항과 동일합니다~!
코드 포맷 자체의 통일성을 위해서도 지속 확인해주세요
```suggestion
   if (user) {
```